### PR TITLE
open-cluster-management/backlog#7554 adding new metric types and OCP …

### PR DESCRIPTION
**Pull request by swopebe**
…info

### CODE REVIEW

File: observability/observe_intro.adoc [hunk 1]
```diff
@@ -27,6 +27,16 @@ You can customize the observability service by creating custom link:https://prom
 
 For more information about enabling observability, see link:../observability/observability_install.adoc#enable-observability[Enable observability service].
 
+[#metric-types]
+=== Metric types
+
+By default, {ocp-short} sends metrics to Red Hat using the Telemetry service. The following additional metrics are available with {product-title-short} and are included with telemetry, but are _not_ displayed on the {product-title-short} Observe environments Overview dashboard:
```
Comment (dockerymick) [inline observability/observe_intro.adoc]: Should "Observe environments Overview" be italicized since it is the name of the dashboard?

**Review by dockerymick — APPROVED** [review]
I left a question for you. All other updates look great, thanks for adding this Brandi

File: observability/observe_intro.adoc (hunk 1 above)
Comment (swopebe) [reply observability/observe_intro.adoc]: yes!

**Review by mark-nc — APPROVED** [review]
/lgtm

**Pull request merged**